src/check-couch-up.js

```diff
@@ -1,5 +1,5 @@
 const utils = require('./utils');
-const NUM_RETRY = 200;
+const NUM_RETRY = 300;
```

Comment:
Is this increase really needed?

Reply:
Yup. Spinning up the second cluster where the nodes are still referring to an old cluster takes a bit more time.